tensorboard/plugins/debugger/README.md

```diff
@@ -0,0 +1,175 @@
+# The Debugger Dashboard
+
+The debugger dashboard offers a graphical user interface for the [TensorFlow debugger](https://www.tensorflow.org/programmers_guide/debugger). For instance, this dashboard enables users to
```

Comment:
Optional: s/The debugger dashboard/This dashboard/

Reply:
This is a cherry-pick of a README written by Chi in #942.  If you think these edits are release-blocking, please send a PR with changes to Chi ASAP and I will cherry-pick it into the release PR once it's merged into master - I'm not editing this in the release PR.  Ditto for all the other comments on this file.